src/firestore/util.ts

```diff
@@ -162,6 +198,17 @@ export const initializeFirebaseProject = async (
     };
   } else {
     const app = safeInitializeApp(config as LiveFirebaseConfig, name);
```

Comment:
But note that if we pass in the `siteKey` and `debugToken` as part of the config, it will also be passed into `safeInitializeApp`. Maybe this is okay. But you could also destruction like so
```
const { siteKey, debugToken, ...appConfig } = config;
const app = safeInitializeApp(appConfig as LiveFirebaseConfig, name);
```